src/model_clay.py

```diff
@@ -287,6 +289,9 @@ def forward(self, datacube):
             cube
         )  # [B G L D] - patchify & create embeddings per patch
 
+        # Move position & band encoding to the device
+        self.pos_encoding = self.pos_encoding.to(patches.device)
+        self.band_encoding = self.band_encoding.to(patches.device)
```

Comment:
Hmm, I thought these `.to()` lines aren't needed anymore after https://github.com/Clay-foundation/model/pull/95#discussion_r1432381686? Are you still getting errors like `RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu`?

Reply:
There were two more lines that were missed, so I removed them in 52243a9a6c96b8b7342ad711cd3211ce5c9ccb80